src/main/java/com/shevtsov/services/Authorisation.java

```diff
@@ -0,0 +1,18 @@
+package com.shevtsov.services;
+
+public interface Authorisation {
+
+    /**
+     * Authorizes the client by entered phone number.
+     * @param phone The phone number of client, which is searched.
+     * @return for now true if client was searched, or false otherwise.
+     * Plan to change the return value to Client-object later.
+     */
+    boolean authorizeClient(String phone);
+
+    /**
+     * Checks the user for matching with the admin.
+     * @return true if the user matches, or false otherwise.
+     */
+    boolean authorizeAdmin();
```

Comment:
How can you define Admin?

Reply:
simple way: an user enters a password, and this method compares it with the standard, returns true or false. it is acceptable?

Reply:
but this method does not accept password